wiki/osu!stream/en.md

```diff
@@ -1,15 +1,15 @@
-<!-- TODO: some updates and needs a proper end of life message. it's also open source now -->
-
 # osu!stream
 
 ![](img/Os-Logo.jpg "osu!stream logo")
 
-**osu!stream** is a version of osu! built from scratch that is made especially for touch-enabled devices. It is currently available for Apple's [iOS](http://en.wikipedia.org/wiki/IOS) platform only, but an [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) port is planned. It introduces a brand new game-play system such as the hold circle and connected beats.
+**osu!stream** is a version of osu! built from scratch that is made especially for touch-enabled devices. It is currently available for Apple's [iOS](http://en.wikipedia.org/wiki/IOS) platform only. The development ended in 2020 and is to be seen as finished. An [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) port was planned but not implemented yet. For more information, read the [blog post](https://blog.ppy.sh/osu-stream-2020-release/) about its last release.
 
-It does not support classic osu! beatmaps. Instead, maps are created by a special team of mappers to take advantage of the new elements and adhere to a stricter level of standards. Both free and paid maps are available at the "Store".
+It introduces a brand new gameplay system such as the hold circle and connected beats. It does not support classic osu! beatmaps. Instead, maps are created by a special team of mappers to take advantage of the new elements and adhere to a stricter level of standards. All maps are available for free at the "Store".
```

Comment:
```suggestion
It introduces a brand new gameplay system such as the hold circle and connected beats. It does not support classic osu! beatmaps without manual conversion. Instead, maps are created by a special team of mappers to take advantage of the new elements and adhere to a stricter level of standards. All maps are available for free at the "Store".
```

Not sure if you want to mention this here, but the conversion tools are provided as part of the open source release, so you can actually play any beatmap if you're willing to put in the effort.

Reply:
Also is it worth saying that some beatmaps were purchasable in the past? Losing a bit of historical context by saying that all beatmaps are free.

Reply:
> It introduced a brand new gameplay system, with new gameplay elements such as hold circles and connected beats. It does not support classic osu! beatmaps without manual conversion (the tools for conversion are provided in the open source release). Instead, maps were created by a special team of mappers to take advantage of the new elements and adhere to a [stricter level of standards](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g). All maps are available for free at the "Store". Previously, some were purchasable as beatmap packs.

How about this one? I think this is a good in-between.

Reply:
```suggestion
It introduced a brand new gameplay system, with new gameplay elements such as hold circles and connected beats. It does not support classic osu! beatmaps without manual conversion (the tools for conversion are provided in the open source release). Instead, maps were created by a special team of mappers to take advantage of the new elements and adhere to a [stricter level of standards](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g).
```